src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -194,7 +195,7 @@ public static unsafe int IndexOf(ref byte searchSpace, byte value, int length)
             nuint offset = 0; // Use nuint for arithmetic to avoid unnecessary 64->32->64 truncations
             nuint lengthToExamine = (nuint)(uint)length;
 
-            if (Avx2.IsSupported || Sse2.IsSupported)
+            if (Avx2.IsSupported || Sse2.IsSupported || AdvSimd.IsSupported)
```

Comment:
Just curious, isn't `Avx2.IsSupported || Sse2.IsSupported` the same as `Sse2.IsSupported`  (my assumption was all machines supporting AVX2 will support SSE2)